cli/RenderMailTemplatesCommand.php

```diff
@@ -0,0 +1,156 @@
+
+		$configReader = new ConfigReader(
+			new SimpleFileFetcher(),
+			__DIR__ . '/../app/config/config.dist.json',
+			is_readable( $prodConfigPath ) ? $prodConfigPath : null
+		);
+
+		return $configReader->getConfig();
+	}
+
+	/**
+	 * Check that there are templates for all fixtures and (even more important) vice-versa
+	 *
+	 * @param array $testData Template names and fixture information to render these templates
+	 * @param array $mailTemplatePaths Directories containing templates
+	 * @param OutputInterface $output Command output
+	 */
+	private function validateTemplateFixtures( array $testData, array $mailTemplatePaths, OutputInterface $output ): void {
+		$mailTemplatesOnDisk = [];
+		foreach ( $mailTemplatePaths as $path ) {
+			$mailFilesInFolder = glob( $path . '/Mail_*' );
+			array_walk( $mailFilesInFolder, function( & $filename ) {
+				$filename = basename( $filename ); // this would cause problems w/ mail templates in sub-folders
+			} );
+			$mailTemplatesOnDisk = array_merge( $mailTemplatesOnDisk, $mailFilesInFolder );
+		}
+
+		$testTemplateNames = array_keys( $testData );
+
+		$untestedTemplates = array_diff( $mailTemplatesOnDisk, $testTemplateNames );
+		if ( count( $untestedTemplates ) ) {
+			$output->writeln(
+				'<error>There are untested templates: ' . implode( ', ', $untestedTemplates ) . '</error>'
+			);
+		}
+
+		$strayTemplates = array_diff( $testTemplateNames, $mailTemplatesOnDisk );
+		if ( count( $strayTemplates ) ) {
+			$output->writeln(
+				'<error>There are tests for non-existing templates: ' . implode( ', ', $strayTemplates ) . '</error>'
+			);
+		}
+	}
+
+	/**
+	 * Render all templates and write them to disk to allow a comparison with an alternative data set
+	 *
+	 * @param array $testData Template names and fixture information to render these templates
+	 * @param Twig_Environment $twig The templating engine to render the templates
+	 * @param string $outputPath Path where rendered templates will be written to
+	 * @param OutputInterface $output Command output
+	 */
+	private function renderTemplates( array $testData, Twig_Environment $twig, string $outputPath, OutputInterface $output ): void {
+		foreach( $testData as $template => $templateSettings ) {
+
+			if ( empty( $templateSettings['variants'] ) ) {
+				$templateSettings['variants'] = [ '' => [] ];
+			}
+
+			foreach( $templateSettings['variants'] as $variantName => $additionalContext ) {
+				$outputName =
+					$outputPath .
+					basename( $template, '.txt.twig' ) .
+					( $variantName ? ".$variantName" : '' ) .
+					'.txt';
+
+				$output->write( "$outputName" );
+				if ( file_exists( $outputName ) ) {
+					$output->writeln( "$outputName already exists, skipping ..." );
+					continue;
+				}
+
+				try {
+					file_put_contents( $outputName, $twig->render( $template, array_merge_recursive(
+						$templateSettings['context'],
+						$templateSettings['variants'][$variantName]
+					) ) );
+				} catch( Twig_Error $e ) {
```

Comment:
There does not seem to be a rule for that (neither followed in the code, nor in phpcs) - 100+ matches inside the project w/o space. It's not much different from a normal function call which we also write w/o space, right?

    vagrant@vagrant:/vagrant$ ./vendor/bin/phpcs -v cli
    Registering sniffs in the FunFunFundraisingFrontend standard... DONE (62 sniffs registered)
    Creating file list... DONE (6 files in queue)
    Changing into directory /vagrant/cli/ConfigValidation
    Processing ConfigValidationException.php [PHP => 61 tokens in 13 lines]... DONE in 40ms (0 errors, 0 warnings)
    Processing SchemaLoader.php [PHP => 284 tokens in 37 lines]... DONE in 8ms (0 errors, 0 warnings)
    Processing ValidationErrorRenderer.php [PHP => 401 tokens in 41 lines]... DONE in 15ms (0 errors, 0 warnings)
    Processing ValidateConfigCommand.php [PHP => 778 tokens in 98 lines]... DONE in 9ms (0 errors, 0 warnings)
    Changing into directory /vagrant/cli
    Processing FundraisingCli.php [PHP => 271 tokens in 39 lines]... DONE in 4ms (0 errors, 0 warnings)
    Processing RenderMailTemplatesCommand.php [PHP => 1297 tokens in 156 lines]... DONE in 25ms (0 errors, 0 warnings)
    Time: 440ms; Memory: 4Mb